Templates/Per Tenant Extension/.github/workflows/CICD.yaml

```diff
@@ -60,8 +58,26 @@ jobs:
         with:
           shell: powershell
           parentTelemetryScopeJson: ${{ steps.init.outputs.telemetryScopeJson }}
-          getProjects: 'Y'
           getEnvironments: '*'
+          
+      - name: Load Build Projects
+        id: loadProjects
+        uses: microsoft/AL-Go-Actions/LoadProjects@main
+        with:
+          shell: powershell
+
+      - name: Verify workflow depth
```

Comment:
😓

I'll try 😄

Reply:
I tried to think of a way, but an action that just compares two numbers seems a bit of an overkill.

Anyway, it can be addressed in a follow-up PR if you feel strongly about it. I'll resolve this comment now.